web-app/js/portal/details/DataCollectionDetailsPanel.js

```diff
@@ -13,7 +13,7 @@ Portal.details.DataCollectionDetailsPanel = Ext.extend(Ext.Panel, {
 
         var tabPanelForLayer = this._initSubsetItemsTabPanel(cfg);
 
-        this.createTools(cfg.dataCollection.getLayerState());
+        this.createTools(cfg.dataCollection.getLayerAdapter());
 
         cfg.dataCollection.getLayerState().on('loadstart', function() {
```

Comment:
Just a question about using the new adapter vs using the state. Why do we listen to layer state events here but use the adapter to check if the layer is loading?